enforcer-rules/pom.xml

```diff
@@ -77,7 +87,13 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
-      <version>2.24.5</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <!-- Used by test to resolve artifacts -->
```

Comment:
scope test then

Reply:
That was incorrect. It's used in main tool.